packages/main/src/plugin/index.ts

```diff
@@ -459,6 +459,10 @@ export class PluginSystem {
 
     const messageBox = new MessageBox(apiSender);
 
+    app.on('before-quit', () => {
```

Comment:
we've already this event handled in this class and we call extension loader stop all extensions

For now, let's move the kubernetesClient dispose to extension loader.stopAllExtensions

We'll add later a stop all that should call stop all extensions and dispose on the kubernetesClient